docs/content/direct/get-started.md

```diff
@@ -1,6 +1,30 @@
 # Getting Started with KubeStellar
 
-This page shows one concrete example of steps 2--7 from the [full Installation and Usage outline](user-guide-intro.md#the-full-story). This example produces a simple single-host system suitable for kicking the tires, using [kind](https://kind.sigs.k8s.io/) to create three new clusters to serve as your KubeFlex hosting cluster and two WECs. This page concludes with forwarding you to one example of the remaining steps.
+There are multiple ways to get started with Kubestellar, reflecting its flexible architecture that supports various deployment patterns. This page presents two approaches:
+
+1. A **Step by step walkthrough** that demonstrates the core concepts and components, showing how to manually set up a simple single-host system.
+
+2. A **quick automated setup** using our installation script, which creates a basic working environment for those who want to start experimenting right away. 
+
+Both approaches create a basic environment suitable for learning, but remember that KubeStellar supports more sophisticated configurations including:
+
+- Multiple Inventory and Transport Spaces (ITS)
+- Multiple Workload Definition Spaces (WDS)
+- Dynamic addition and removal of Workload Execution Clusters (WECs)
+- Various deployment patterns to suit different organizational needs
+
+## Quick Start Using the Automated Script
+
+If you want to quickly setup a basic environment, you can use our automated installation script:
+
```

Comment:
I think it would be worthwhile to point here to the pre-requisites section, which is a little more expansive and helpful than the output of the checking script.

If we believe that the note for Windows users should stay, it would apply to both techniques, right?